app/views/contexts/section_summary_context.py

```diff
@@ -171,10 +189,8 @@ def _list_summary_element(self, summary, current_location, section) -> Mapping:
             ),
         }
 
-    def _add_link(self, summary, current_location, section, list_collector_block):
-        routing_path = self._router.routing_path(
-            section["id"], current_location.list_item_id
-        )
+    @staticmethod
+    def _add_link(summary, section, list_collector_block, routing_path):
 
         if list_collector_block["id"] in routing_path:
             return url_for(
```

Comment:
Once my comment about removing routing path from this method is done, you can update this to:
```py
        if list_collector_block:
            return url_for(
```

Reply:
Done👍